components/ActualGamesList/ActualGame/ActualGame.tsx

```diff
@@ -11,6 +13,16 @@ export const ActualGame: React.FC<ActualGamePropsI> = ({
   proposer,
   rules,
 }) => {
+  const [ticTacToeAddress, setTicTacToeAddress] = useState('')
+  const [checkersAddress, setCheckersAddress] = useState('')
+
+  getRulesContract('tic-tac-toe').then(function(response) {
+    setTicTacToeAddress(response.address.toLowerCase())
+  })
+  getRulesContract('checkers').then(function(response) {
```

Comment:
What if we have more than two game types? Should we write an additional separated state and a handler for each of them?